borg/archiver.py

```diff
@@ -294,16 +296,16 @@ def create_inner(archive, cache):
 
     def _process(self, archive, cache, matcher, exclude_caches, exclude_if_present,
                  keep_tag_files, skip_inodes, path, restrict_dev,
-                 read_special=False, dry_run=False):
+                 read_special=False, dry_run=False, st=None):
         if not matcher.match(path):
             self.print_file_status('x', path)
             return
-
-        try:
-            st = os.lstat(path)
-        except OSError as e:
-            self.print_warning('%s: %s', path, e)
-            return
+        if not st:
```

Comment:
this can go miserably wrong (maybe not in this case, but e.g. if len(sequence_object) == 0.

so better get used to "is None" if you mean it. ;)